controllers/spec/common.go

```diff
@@ -1114,7 +1112,7 @@ func parseJavaLogLevel(runtime *v1alpha1.JavaRuntime) string {
 		v1alpha1.LogLevelFatal: "FATAL",
 		v1alpha1.LogLevelOff:   "OFF",
 	}
-	if runtime.Log != nil && runtime.Log.Level != "" && runtime.Log.LogConfig == nil {
+	if runtime.Log != nil && runtime.Log.Level != "" {
```

Comment:
for java runtime, we should always get the log level and pass it as the argument: `-Dpulsar.log.level=`